docs/tutorials/install_and_admin/geonode_install/install_geonode_application.txt

```diff
@@ -90,14 +90,14 @@ GeoNode Setup
 
 Let's download GeoNode from the main GeoNode repository on GitHub:
 
-.. 
+..
     For the purpose of this training the GeoNode repository has
     been downloaded and placed in the `geonode` user `home folder` (/home/geo/geonode).
     *Skip* the following command group.
 
 ::
 
-    git clone https://github.com/GeoNode/geonode.git
+    git clone https://github.com/GeoNode/geonode.git --branch 2.6.x
```

Comment:
Not really. It was according to [this answer ](http://osgeo-org.1560.x6.nabble.com/Error-creating-superuser-gn2-4-on-Debian-tp5276644p5276725.html) you gave me recently. I thought these changes would be deployed on v2.6.x. Should I fix it??